examples/joke_bot/src/main.rs

```diff
@@ -99,7 +99,7 @@ async fn run() -> Result<(), Box<dyn Error>> {
             }
         });
 
-        let invite_futures = response.rooms.invite.iter().map(|(room_id, _)| async move {
+        let invite_futures = response.rooms.invite.keys().map(|room_id| async move {
```

Comment:
was this a result of clippy complaining?

Reply:
Yeah